src/sensai/lightgbm/lightgbm.py

```diff
@@ -0,0 +1,60 @@
+from typing import Sequence
+import logging
+import lightgbm
+import pandas as pd
+
+from sensai.sklearn.sklearn_base import AbstractSkLearnMultipleOneDimVectorRegressionModel, \
```

Comment:
Never use absolute imports within the library.

Reply:
Done